lib/lasso-require-plugin.js

```diff
@@ -167,7 +190,10 @@ module.exports = exports = function plugin(lasso, config) {
 
         lasso.dependencies.registerRequireExtension(babelExtension, {
             read: function(path, lassoContext, callback) {
-                babelTransformFile(path, callback);
+                if (isPathWhitelisted(path)) {
+                    return babelTransformFile(path, callback);
+                }
+                return fs.createReadStream(path, { encoding: 'utf8' });
```

Comment:
@patrick-steele-idem not sure if this line is needed, just added it if the path was not whitelisted